### PR TITLE
[go/zh-cn] Update content from English fork

### PR DESCRIPTION
**Pull request by ttimasdf**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

**Comment by vendethiel** [comment]
can you please translate strings?

**Comment by ttimasdf** [comment]
Do you mean the strings in codes like these?
```go
    fmt.Println(sentenceFactory("summer")("A beautiful", "day!"))
```

**Comment by vendethiel** [comment]
Yes, please.

**Comment by ttimasdf** [comment]
check it out 😄

**Comment by vendethiel** [comment]
Thanks!

File: zh-cn/go-cn.html.markdown [hunk 1]
```diff
@@ -36,7 +40,7 @@ import (
 func main() {
     // 往标准输出打印一行。
     // 用包名fmt限制打印函数。
-    fmt.Println("Hello world!")
+    fmt.Println("天坑欢迎你!")
```
Comment (aisk) [inline zh-cn/go-cn.html.markdown:43]: I think this is not suitable, these words have negative meanings.

Comment (vendethiel) [reply zh-cn/go-cn.html.markdown:43]: please send a new PR then

**Pull request merged**